base/db/tests/testthat/test.check.missing.files.R

```diff
@@ -0,0 +1,19 @@
+test_that("`check_missing_files()` able to return correct missing files", {
```

Comment:
Filename should match function name (so use `_` not `.`)
Come to think of it, same for the files containing the function definitions

Reply:
Fixed